app/locale/fi.coffee

```diff
@@ -3050,7 +3050,7 @@ module.exports = nativeDescription: "suomi", englishDescription: "Finnish", tran
 #    learn_at_own_pace: "Learn at Your Own Pace"
 #    monthly_sub: "Monthly Subscription"
 #    buy_now: "Buy now"
-#    per_month: " / mo"
+    per_month: " / kk" 
```

Comment:
if you want to enable this key, its parent which is `parents_landing_2` needs to be uncommented by removing `#`